stdlib/3/collections/__init__.pyi

```diff
@@ -235,6 +236,8 @@ class deque(MutableSequence[_T], Generic[_T]):
     def __contains__(self, o: object) -> bool: ...
     def __reversed__(self) -> Iterator[_T]: ...
 
+    def __iadd__(self: _S, iterable: Iterable[_T]) -> _S[_T]: ...
```

Comment:
mypy complains about the subscripted `_S[_T]`. In fact, this is not necessary and just `_S` should be sufficient, since `self` already has the correct type.

Reply:
Thanks. It surprises me that I didn't get a complaint with mypy 0.660 locally. Oh well.

I also added type annotation for Python 2's `collections.deque.__iadd__` for good measure.